backend/Pipfile

```diff
@@ -40,7 +40,7 @@ boto3 = "*"
 apns2 = "*"
 numpy = "*"
 click = "==8.0.4"
-celery = "5"
```

Comment:
Looks good! Per discussion, this is because there was no reason to have celery be hard-set to 5.0. 

Also, let's remove this from the other PR since we are making the change here.